twiml/twiml.go

```diff
@@ -0,0 +1,81 @@
+package twiml
+
+import (
+	"github.com/beevik/etree"
+	"strings"
+)
+
+type Noun interface {
+	GetName() string
+	GetText() string
+	GetAttr() (map[string]string, map[string]string)
+}
+
+type Verb interface {
+	GetName() string
+	GetText() string
+	GetAttr() (map[string]string, map[string]string)
+	GetNouns() []Noun
+}
+
+func AddAllVerbs(response *etree.Element, verbs []Verb) {
+	for _, verb := range verbs {
+		verbEl := CreateVerbElement(verb)
+		response.AddChild(verbEl)
+	}
+}
+
+func CreateVerbElement(verb Verb) *etree.Element {
```

Comment:
Should make anything in here that's not needed as public into private (e.g., *c*reateVerbElement)